modules/measure/src/main/java/com/opengamma/strata/measure/fxopt/BlackFxOptionSmileVolatilitiesSpecification.java

```diff
@@ -273,20 +273,20 @@ private BlackFxOptionSmileVolatilitiesSpecification(
       ImmutableList<FxOptionVolatilitiesNode> nodesForTenor = nodesByTenor.get(tenor);
       // value type, delta, size
       List<Double> atmDelta = nodesForTenor.stream()
-          .filter(n -> n.getQuoteValueType().equals(ValueType.BLACK_VOLATILITY))
-          .map(n -> n.getStrike().getValue())
+          .filter(node -> node.getQuoteValueType().equals(ValueType.BLACK_VOLATILITY))
+          .map(node -> node.getStrike().getValue())
           .sorted()
           .collect(toList());
       ArgChecker.isTrue(atmDelta.equals(fullDeltas.subList(nDeltas, nDeltas + 1)));
       List<Double> rrDelta = nodesForTenor.stream()
-          .filter(n -> n.getQuoteValueType().equals(ValueType.RISK_REVERSAL))
-          .map(n -> n.getStrike().getValue())
+          .filter(node -> node.getQuoteValueType().equals(ValueType.RISK_REVERSAL))
+          .map(node -> node.getStrike().getValue())
           .sorted()
           .collect(toList());
       ArgChecker.isTrue(rrDelta.equals(deltas));
       List<Double> strDelta = nodesForTenor.stream()
-          .filter(n -> n.getQuoteValueType().equals(ValueType.STRANGLE))
-          .map(n -> n.getStrike().getValue())
+          .filter(node -> node.getQuoteValueType().equals(ValueType.STRANGLE))
+          .map(node -> node.getStrike().getValue())
```

Comment:
FYI, not every lambda parameter needs to be a word, just where it is confusing ;-)